timify-ui/src/components/EpisodeDetail/EpisodeDetail.jsx

```diff
@@ -7,14 +7,12 @@ const EpisodeDetail = ({episode}) => {
   return (
     <Link to={`/podcast/episode/${episode.uuid}`}>
     <div onClick={() => (episode)}>
-        <div key={episode.uuid} className='episode-each'>
-        <p>{episode.name}</p>
-        <p>{episode.episodeNumber}</p>
-        <p>{episode.datePublished}</p>
```

Comment:
why are we getting rid of the date?

Reply:
i

> why are we getting rid of the date?

it comes out in a really weird format from the API. it doesn't look like a normal mm-dd-yyyy